### PR TITLE
Multi bracket validation

### PR DESCRIPTION
**Pull request by daesy13**

### CODE REVIEW

**Comment by daesy13** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

File: code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+import java.util.Stack;
+
 public class MultiBracketValidation {
-    // WORK IN PROGRESS
+
+    public static void main(String[] args){
+        System.out.println(multiBracketValidation("{}{Code}[Fellows](())"));
+        System.out.println(multiBracketValidation("[({}]"));
+        System.out.println(multiBracketValidation("(]("));
+    }
```
Comment (jeffborda) [inline code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java]: Delete this main method... we don't want main methods in our Library classes. It's faster to just write a quick test for these things.

File: code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java [hunk 2]
```diff
@@ -1,4 +1,29 @@
...
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
 public class MultiBracketValidationTest {
+// HAPPY TEST
+    @Test public void allCorrectOrderTest(){
+        String input = "{}{Code}[Fellows](())";
+        boolean actual = multiBracketValidation(input);
+        assertTrue(actual);
+    }
```
Comment (jeffborda) [inline code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java]: I would get rid of the variable you're using to store the boolean results from the method call and just do that in the assert statement. It would look like this: `assertTrue(multiBracketValidation(input));`

File: code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java [hunk 3]
```diff
@@ -1,4 +1,29 @@
...
+// FAIL CASE
+    @Test public void failCaseTest(){
+        String input = "(";
+        boolean actual = multiBracketValidation(input);
+        System.out.println(actual);
+        assertFalse(actual);
+    }
```
Comment (jeffborda) [inline code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java]: This whole test could really just be a one liner: `assertFalse(multiBracketValidation("("));`

File: code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java [hunk 4]
```diff
@@ -1,4 +1,29 @@
...
+// EDGE CASE
+    @Test public void edgeCaseTest(){
+        String input = "[({}]";
+        boolean actual = multiBracketValidation(input);
+        assertFalse(actual);
+    }
```
Comment (jeffborda) [inline code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java]: Same here, too many variables... just do: `asserFalse(multiBracketValidation("[({}]"));`
